internal/manager/controllerdef.go

```diff
@@ -255,7 +255,7 @@ func setupControllers(
 		// Gateway API Controllers - Beta APIs
```

Comment:
```suggestion
		// Gateway API Controllers
```
they won't be beta for gwapi 1.0, right? We can adjust it now I think